### PR TITLE
Doc page for Table component

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

File: website/screens/components/table/specs/TableSpecsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+            text="select"
+            newWindow
+          />
+          . For concrete specifications about states, please, consider to see
+          the documentation of each component.
+        </DxcText>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Anatomy</HeadingLink>
+        <Figure caption="Table component anatomy">
```
Comment (GomezIvann) [inline website/screens/components/table/specs/TableSpecsPage.tsx:52]: Anatomy Images don't need a `Figure` surrounding them, we should remove it.

File: website/screens/components/table/specs/TableSpecsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+              <th>Component token</th>
+              <th>Element</th>
+              <th>Core token</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <code>dataBackgroundColor</code>
```
Comment (GomezIvann) [inline website/screens/components/table/specs/TableSpecsPage.tsx:80]: Change each `<code>` to our custom `<Code>`.

File: website/screens/components/table/specs/TableSpecsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+          <tbody>
+            <tr>
+              <td>
+                <code>dataFontFamily</code>
+              </td>
+              <td>Cell value</td>
+              <td>
+                <code>font-family-sans</code>
+              </td>
+              <td>&#39;Open Sans&#39;, sans-serif</td>
```
Comment (GomezIvann) [inline website/screens/components/table/specs/TableSpecsPage.tsx:179]: Should we change these codes to actual apostrophes?

File: website/screens/components/table/specs/TableSpecsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+            </tr>
+            <tr>
+              <td>
+                <code>headerFontFamily</code>
+              </td>
+              <td>Header title</td>
+              <td>
+                <code>font-family-sans</code>
+              </td>
+              <td>&#39;Open Sans&#39;, sans-serif</td>
```
Comment (GomezIvann) [inline website/screens/components/table/specs/TableSpecsPage.tsx:243]: Same as above.

**Review by GomezIvann — CHANGES_REQUESTED** [review]

File: website/screens/components/table/specs/TableSpecsPage.tsx (hunk 1 above)
Comment (Lisa159) [reply website/screens/components/table/specs/TableSpecsPage.tsx]: Done.

File: website/screens/components/table/specs/TableSpecsPage.tsx (hunk 4 above)
Comment (Lisa159) [reply website/screens/components/table/specs/TableSpecsPage.tsx]: Done.

File: website/screens/components/table/specs/TableSpecsPage.tsx (hunk 3 above)
Comment (Lisa159) [reply website/screens/components/table/specs/TableSpecsPage.tsx]: Done.

**Review by GomezIvann — CHANGES_REQUESTED** [review]

File: website/screens/components/table/specs/TableSpecsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,542 @@
...
+          <Image src={specsImage} alt="Table design specifications" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>States</HeadingLink>
+        <DxcText as="p">
+          Some other components defined in the design system are used in the
+          table component as the{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/checkbox"
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/table/specs/TableSpecsPage.tsx]: Done.

File: website/screens/components/table/specs/TableSpecsPage.tsx [hunk 6]
```diff
@@ -0,0 +1,542 @@
...
+            text="checkbox"
+            newWindow
+          />
+          ,{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/button"
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/table/specs/TableSpecsPage.tsx]: Done.

File: website/screens/components/table/specs/TableSpecsPage.tsx [hunk 7]
```diff
@@ -0,0 +1,542 @@
...
+            text="button"
+            newWindow
+          />{" "}
+          or{" "}
+          <DxcLink
+            href="https://developer.dxc.com/design/guidelines/components/select"
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/table/specs/TableSpecsPage.tsx]: Done.

**Review by GomezIvann — APPROVED** [review]

**Pull request merged**